synapse/lib/node.py

```diff
@@ -254,7 +254,7 @@ def get(self, name):
         '''
         parts = name.split('::', 1)
 
-        if len(parts) is 1:
+        if len(parts) == 1:
```

Comment:
this is *technically* okay as long as we're using cpython

Reply:
actually we should probably gut this parts splitting since it was used for the now-dead implicit pivot which is moved to to AST